system/Config/Factories.php

```diff
@@ -253,7 +257,8 @@ public static function getOptions(string $component): array
      *
      * @param string $component Lowercase, plural component name
      *
-     * @return array<string, mixed> The result after applying defaults and normalization
+     * @return array The result after applying defaults and normalization
+     * @phpstan-return array<string, string|null|bool>
```

Comment:
Should `@return` be the same with `@phpstan-return`?

Reply:
Done.